public/scripts/wiki.js

```diff
@@ -50,7 +52,6 @@ var WikiBox = React.createClass({
 
     return (
       <div className="wiki-box">
-        <Navbar />
```

Comment:
I realized it needed to be linked with content, so I shifted it down. We can either just leave it like that or combine article box and wiki-box